snappiershot/plugins/pytest.py

```diff
@@ -60,6 +61,22 @@ def pytest_configure(config: PytestConfig) -> None:
         setattr(config.option, PACKAGE_TRACKER_OPTION, SnapshotTracker(*test_paths))
 
 
+def pytest_terminal_summary(
+    terminalreporter: TerminalReporter, config: PytestConfig
+) -> None:
+    """ Add a summary to the end of the of the pytest output about snapshot statuses. """
+    tracker: SnapshotTracker = config.getoption(PACKAGE_TRACKER_OPTION)
+    status_report = tracker.get_status_report()
+
+    if status_report.any():
```

Comment:
Do we want this `if` clause? (e.g. it wouldn't print if all `0`s)

Reply:
I added it because I was seeing it print out for our own tests, and figured that the additional check here wouldn't hurt